### PR TITLE
Update default arm api version, update ARM package, add test for CrossRG

### PR DESCRIPTION
**Pull request by vivsriaus**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by yugangw-msft** [comment]
@vivsriaus, please update the yaml files
Due to time constraint, you can manually update other team's yaml files to use the new api-version on creating RG. For other mismatches, please investiagate

File: src/command_modules/azure-cli-resource/tests/test_resource.py [hunk 1]
```diff
@@ -656,5 +656,39 @@ def test_managedapp(self, resource_group):
...
+        self.cmd('group deployment show -g {} -n {}'.format(resource_group, deployment_name),
+                 checks=[
+                     JCheck('name', deployment_name),
+                     JCheck('resourceGroup', resource_group)
+                 ])
+        self.cmd('group deployment operation list -g {} -n {}'.format(resource_group,
+                                                                      deployment_name), checks=[
+            JCheck('length([])', 3),
+            JCheck('[0].resourceGroup', resource_group)
+        ])
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-resource/tests/test_resource.py]: I think you will need to delete the  `crossrg4`

**Comment by yugangw-msft** [comment]
LGTM. Please get CI green, and log an issue on our testfx for options to skip api-version matching

File: src/command_modules/azure-cli-resource/tests/test_resource.py (hunk 1 above)
Comment (vivsriaus) [inline src/command_modules/azure-cli-resource/tests/test_resource.py]: Addressed

**Pull request merged**